ui/webui/test/check-installation-review

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/python3
+#
+# Copyright (C) 2022 Red Hat, Inc.
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of the GNU Lesser General Public License as published by
+# the Free Software Foundation; either version 2.1 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with this program; If not, see <http://www.gnu.org/licenses/>.
+
+import os
+import sys
+
+# import Cockpit's machinery for test VMs and its browser test API
+TEST_DIR = os.path.dirname(__file__)
+sys.path.append(os.path.join(TEST_DIR, "common"))
+sys.path.append(os.path.join(os.path.dirname(TEST_DIR), "bots/machine"))
+
+from testlib import MachineCase, nondestructive, test_main  # pylint: disable=import-error
+from machine_install import VirtInstallMachine
+
+@nondestructive
+class TestReview(MachineCase):
+    MachineCase.machine_class = VirtInstallMachine
+
+    def testBasic(self):
+        b = self.browser
+
+        b.open("/cockpit/@localhost/anaconda-webui/index.html#/installation-language")
+
+        for page in ['installation-language', 'installation-destination']:
+            # with the pages basically empty of common elements (as those are provided by the top-level wizard widget)
+            # we at least iterate over them to check this works as expected
+            b.click("button.pf-m-primary")
```

Comment:
You can now use the helper classes, by doing:

i = Installer(b)

i.next()

Reply:
See how I import these in the other files.

Reply:
Oh, right, missed that, works now. :)